### DISCUSSION TITLE
agent-handbook/docs/agent-intro/memory-of-agent

### DISCUSSION THREAD

**Discussion by giscus** [wj-Mcat/agent-handbook#5] [General]
# agent-handbook/docs/agent-intro/memory-of-agent

说的高级一点，就是智能体拥有长期记忆；说的Low 一点，就是把历史对话和开发者觉得重要的相关信息以一定格式塞入到 Prompt 当中，这样大模型就能够看到历史信息，此时便拥有了记忆。

https://wj-mcat.github.io/agent-handbook/docs/agent-intro/memory-of-agent

**Comment by wj-Mcat** [comment]
Memory usage
